src/Tickets/Commerce/Gateways/Stripe/Hooks.php

```diff
@@ -99,10 +99,6 @@ public function process_async_stripe_webhook( int $order_id ): void {
 			return;
 		}
 
-		if ( time() < $order->on_checkout_hold ) {
-			return;
-		}
```

Comment:
Without this check here it will still execute if the is a hold on the order I want to ensure it doesnt happen.